src/Apps/Artwork/Components/__stories__/OtherAuctions.story.tsx

```diff
@@ -1,46 +1,45 @@
-import React from "react"
+import { OtherAuctionsQuery } from "__generated__/OtherAuctionsQuery.graphql"
+import { SystemContext } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import React, { useContext } from "react"
+import { graphql, QueryRenderer } from "react-relay"
 import { storiesOf } from "storybook/storiesOf"
 import { Section } from "Utils/Section"
-import { OtherAuctions } from "../OtherAuctions"
+import { OtherAuctionsFragmentContainer } from "../OtherAuctions"
 
-const auctions = [
-  {
-    src: "https://picsum.photos/200/180/?random",
-    headline: "Sotheby’s",
-    subHeadline: "Contemporary Day Sale",
-    badge: "In progress",
-    href: "#",
-  },
-  {
-    src: "https://picsum.photos/400/180/?random",
-    headline: "Sotheby’s",
-    subHeadline: "Contemporary Day Sale",
-    badge: "In progress",
-    href: "#",
-  },
-  {
-    src: "https://picsum.photos/200/600/?random",
-    headline: "Sotheby’s",
-    subHeadline: "Contemporary Day Sale",
-    badge: "In progress",
-    href: "#",
-  },
-  {
-    src: "https://picsum.photos/200/180/?random",
-    headline: "Sotheby’s",
-    subHeadline: "Contemporary Day Sale",
-    badge: "In progress",
-    href: "#",
-  },
-]
+const OtherAuctions = ({ size }: { size?: number }) => {
+  const { relayEnvironment } = useContext(SystemContext)
 
-storiesOf("Apps/Artwork Page/Components/OtherAuctions", module).add(
-  "Other Auctions",
-  () => {
+  return (
+    <QueryRenderer<OtherAuctionsQuery>
+      environment={relayEnvironment}
+      query={graphql`
+        query OtherAuctionsQuery($size: Int!) {
+          sales(size: $size, sort: TIMELY_AT_NAME_ASC) {
+            ...OtherAuctions_sales
+            # Relay isn't transitively expanding the child fragments for some reason, so we'll expand manually.
```

Comment:
Here, because you are importing and using the fragment container (and not the React component), for `OtherAuctions`, you shouldn't need the `AuctionCard_sale` or the `as any` below.